Sources/SwiftKafka/KafkaAcknowledgedMessageError.swift

```diff
@@ -62,3 +62,11 @@ public struct KafkaAcknowledgedMessageError: Error, CustomStringConvertible {
         )
     }
 }
+
+// MARK: - KafkaAcknowledgedMessageError + Hashable
+
+extension KafkaAcknowledgedMessageError: Hashable {}
+
+// MARK: - KafkaAcknowledgedMessageError + Equatable
+
+extension KafkaAcknowledgedMessageError: Equatable {}
```

Comment:
`Hashable` implies `Equatable`

Reply:
Thanks for pointing that out, also removed redundant annotations of `Equatable` for other types that we have